src/components/Insight/InsightEditor/InsightEditor.js

```diff
@@ -105,8 +109,9 @@ class InsightEditor extends Component {
   )
 
   render () {
-    const { id, title, tags, updatedAt, isUpdating, publishDraft } = this.props
+    const { id, title, updatedAt, isUpdating, publishDraft } = this.props
     const { isEditing } = this.state
+    const tags = [...new Set([...this.props.tags, ...this.state.tags])]
```

Comment:
`this.props.tags` will change during the component lifetime?

Reply:
yep. very often. I guess, it's because we use `[trends] SELECTED_WORDS_CLEAR` when draft insight in editing. It's (as I see) for our trends page, where we chose words